pandas/io/excel/_base.py

```diff
@@ -598,6 +598,7 @@ def parse(
                     skiprows=skiprows,
                     nrows=nrows,
                     na_values=na_values,
+                    skip_blank_lines=False,
```

Comment:
Why not expose this parameter in `read_excel` ?

Reply:
We tried that (#40095) but the behaviour when `skip_blank_lines=True` isn't very useful---it only skips empty spreadsheet rows if the entire spreadsheet is contained in column A. The original intent of `skip_blank_lines` was to skip `\n` lines in CSV files, but not lines with empty data elements such as `""\n` or `,,,,\n`, so there isn't an equivalent feature for spreadsheets.

Reply:
I see.  Can make a comment about this and reference your original PR.

Reply:
For sure. Do you mean an inline comment in the code, or in the description of this pull request.